test/transforms/test_line_digraph.py

```diff
@@ -0,0 +1,35 @@
+import torch
+
+from torch_geometric.data import Data
+from torch_geometric.transforms import LineDiGraph
+
+
+def test_line_digraph():
+    transform = LineDiGraph()
+    assert str(transform) == 'LineDiGraph()'
+
+    # Directed.
+    edge_index = torch.tensor([
+        [0, 1, 2, 2, 3],
+        [1, 2, 0, 3, 0],
+    ])
+    data = Data(edge_index=edge_index, num_nodes=4)
+    data = transform(data)
+    assert data.edge_index.tolist() == [[0, 1, 1, 2, 3, 4], [1, 2, 3, 0, 4, 0]]
+    assert data.num_nodes == data.edge_index.max().item() + 1
+
+    # Undirected.
```

Comment:
This is not undirected, and based on the implem, it wouldn't work for an undirected graph

Reply:
Whoops, I started with the existing test_line_graph.py code as a template. I updated the test cases, but must've forgotten to clean up the comments. Thanks.